.gitignore

```diff
@@ -9,4 +9,5 @@
 .idea_modules/
 
 # Visual Code Studio
-.vscode
\ No newline at end of file
+.vscode
+*.sh
```

Comment:
is ignoring shell files on root folder a need for other users ? if not I would recommend to make ignoring only locally using `.git/info/exclude`